Runtime/UIElements/Controls/LoadingSpinner.cs

```diff
@@ -0,0 +1,90 @@
+﻿using JetBrains.Annotations;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace StansAssets.Foundation.UIElements
+{
+    /// <summary>
+    /// The LoadingSpinner control. let's you place buttons group with the labels or images
+    /// </summary>
+    public sealed class LoadingSpinner : VisualElement
+    {
+        [UsedImplicitly]
+        internal new class UxmlFactory : UxmlFactory<LoadingSpinner, UxmlTraits> { }
+
+        internal new class UxmlTraits : BindableElement.UxmlTraits { }
+
+        /// <summary>
+        /// Returns true if control is visible and updating it's progress
+        /// </summary>
+        public bool Started { get; private set; }
+
+        readonly IVisualElementScheduledItem m_ScheduledUpdate;
+        int m_RotationAngle;
+
+        const long k_RotationUpdateInterval = 1L;
+        const int k_RotationAngleDelta = 10;
+
+        /// <summary>
+        /// Loading Spinner control Uss class name
+        /// </summary>
+        public const string UssClassName = "stansassets-loading-spinner";
+
+        /// <summary>
+        /// Creates LoadingSpinner control
+        /// </summary>
+        public LoadingSpinner()
+        {
+            styleSheets.Add( Resources.Load<StyleSheet>("LoadingSpinner"));
+            AddToClassList(UssClassName);
+
+            Started = false;
+            style.display = DisplayStyle.None;
```

Comment:
This feels like something that may create an issue when using the UI Builder.

I am not sure why do we need all the additional API like Start / Stop. I can't imagine the case when you would need "stopped" but visible loading spinner.

Suggestion: Let's start the spinner when VE is attached to the planned and stop the animation when it's deattached.
And let the user to pick show/hide approach by himself.
For example as the user I can do my show hide spinner logic with:

- Add / Remove from the panel
- `style.display`
- `style.visibility`